### PR TITLE
Added large topology tests for static policy in CPU Manager.

### PR DESCRIPTION
**Pull request by balajismaniam**
**What this PR does / why we need it**: This PR adds a very large topology test case for the CPU Manager feature.

Related to #51180. 

CC @ConnorDoyle

### CODE REVIEW

File: pkg/kubelet/cm/cpumanager/policy_static_test.go [hunk 1]
```diff
@@ -227,6 +243,86 @@ func TestStaticPolicyAdd(t *testing.T) {
 			expCPUAlloc:     false,
 			expCSet:         cpuset.NewCPUSet(),
 		},
+		{
+			// All the CPUs from Socket 0 are available. Some CPUs from each
+			// Socket have been already assigned.
+			// Expect all CPUs from Socket 0.
```
Comment (ConnorDoyle) [inline pkg/kubelet/cm/cpumanager/policy_static_test.go]: These descriptive comments are super helpful, thanks for including them.

**Review by ConnorDoyle — APPROVED** [review]
/approve

will add LGTM when last gates turn green

**Comment by ConnorDoyle** [comment]
@derekwaynecarr could you please add to the v1.8 milestone? This is a unit test for a new alpha feature in v1.8.

**Comment by derekwaynecarr** [comment]
added to 1.8 milestone, more testing for stuff, so its a good thing.

**Pull request merged**